synapse/cortex.py

```diff
@@ -102,6 +103,48 @@ def getModelDict(self):
         '''
         return self.cell.model.getModelDict()
 
+    def getCoreInfo(self):
+        '''
+        Return generic information about the cortex including model definition
+        '''
+        return self.cell.getCoreInfo()
+
+    async def addTrigger(self, condition, query, *, info):
+        '''
+        Adds a trigger to the cortex
+        '''
+        username = None if self.user is None else self.user.name
+        self.cell.triggers.add(username, condition, query, info=info)
```

Comment:
Do we want to have a permission (like we have for feed data) about what users may add triggers?

Reply:
That's valid. I think that's best to kick into a separate task.

Reply:
(Added)